<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogView</class>
 <widget class="QWidget" name="LogView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="2">
    <widget class="QWidget" name="nodeListWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="nodeLogWidget" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="logTextWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
